src/redux/walletconnect.ts

```diff
@@ -512,15 +512,19 @@ const listenOnNewMessages = (walletConnector: WalletConnect) => (
       { error, payload },
       logger.DebugContext.walletconnect
     );
+
     if (error) {
       analytics.track('Error on wc call_request', {
         // @ts-ignore
         error,
         payload,
       });
-      logger.error(new RainbowError('WC: Error on wc call_request'));
-      throw error;
+      logger.error(new RainbowError('WC: Error on wc call_request'), {
+        message: error,
+      });
+      return;
```

Comment:
This error only exist a couple dozen times in Sentry, so this can't be the culprit, but we shouldn't need to throw here either.